### PR TITLE
Added Vendor specific rules

### PR DESCRIPTION
**Pull request by gvarao**

### CODE REVIEW

File: community_supplied/VendorSpecific/Cisco/IOS_XR/check-ios-xr-processes-cpu-netconf.rule [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    topic system.processes {
+        rule check-ios-xr-processes-cpu-netconf {
+            keys [ pid process-name ];
+            synopsis "System processes cpu analyzer";
+            description "Collects system process's cpu utilization periodically and notifies anomalies when daemon CPU usage exceed threshold";
+            sensor proc {
+                iAgent {
+                    file XRCpuProcessesTable.yml;
+                    table XRCpuProcessesTable;
+                    frequency 60s;
```
Comment (ConstrainedRandomVar) [inline community_supplied/VendorSpecific/Cisco/IOS_XR/check-ios-xr-processes-cpu-netconf.rule:15]: Can you please make this an offset?

File: community_supplied/VendorSpecific/Paloalto/check-paloalto-processes-cpu-memory-netconf.rule [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+            /*
+             * Sensor configuration to get data from network devices.
+             */
+            sensor systemprocexttable {
+                synopsis "System processes CPU iAgent sensor definition";
+                description "iAgent netconf rpc/cli command sensor to collect data from network device";
+                iAgent {
+                    file PaloaltoSysytemResource.yml;
+                    table PaloaltoSysytemResourceTable;
+                    frequency 60s;
```
Comment (ConstrainedRandomVar) [inline community_supplied/VendorSpecific/Paloalto/check-paloalto-processes-cpu-memory-netconf.rule:51]: And this..

**Review by ConstrainedRandomVar — COMMENTED** [review]
Lookd good. Can you please convert to offsets?

File: community_supplied/VendorSpecific/Cisco/IOS_XR/check-ios-xr-processes-cpu-netconf.rule (hunk 1 above)
Comment (gvarao) [reply community_supplied/VendorSpecific/Cisco/IOS_XR/check-ios-xr-processes-cpu-netconf.rule:15]: Offset is only for trigger frequency and time-range and not for sensor in rule. For sensor, we need to configure under ingest settings.

File: community_supplied/VendorSpecific/Paloalto/check-paloalto-processes-cpu-memory-netconf.rule (hunk 2 above)
Comment (gvarao) [reply community_supplied/VendorSpecific/Paloalto/check-paloalto-processes-cpu-memory-netconf.rule:51]: Offset is only for trigger frequency and time-range and not for sensor in rule. For sensor, we need to configure under ingest settings.

**Review by ConstrainedRandomVar — APPROVED** [review]
My bad on the previous comment for moving to frequency offset. Didn't realize that the trigger and time range portions were already done.. Looks great.

**Pull request merged**